.readthedocs.yaml

```diff
@@ -19,11 +19,14 @@ formats:
 
 # Optionally set the version of Python and requirements required to build your docs
 python:
-  version: 3.8
   install:
     - requirements: doc/requirements.txt
     - method: pip
       path: .
 
 build:
-  image: latest
\ No newline at end of file
+  os: ubuntu-22.04
+  tools:
+    python: "3.9"
+  apt_packages:
+    - graphviz
```

Comment:
Is this a standard update for all plugins?

Reply:
pretty much. PennyLaneAI/pennylane#4086 was needed to fix an SSL import issue, and PennyLaneAI/pennylane#4284 was needed to fix a Graphviz issue caused by the first PR. it's not reaaaally needed here, but I'd rather be consistent everywhere